ConsoleApp_Task1/Program.cs

```diff
@@ -82,44 +34,68 @@ public static double ReturnRadiusOrSide()
                             {
                                 InputString = InputString.Replace(",", ".");
                             }
-                            else
-                            {
-
-                            }
-                            double RadiusOrSideTest = Convert.ToDouble(InputString);
-                            var CircleOrArea = new Figure(RadiusOrSideTest) { };
-                            Console.WriteLine("\nInput Value is " + CircleOrArea.input);
-                            return CircleOrArea.input;
+                            _input = Convert.ToDouble(InputString);
+                            Console.WriteLine("Input Value is " + _input);
+                            return _input;
                         }
                     }
-
                     catch (SystemException sex)
                     {
                         // this class's error string
                         string LastError = sex.Message;
-                        Console.WriteLine("\nIncorrect Input: value is not parsed correctly");
+                        Console.WriteLine("Incorrect Input: value is not parsed correctly");
                         e++;
                     }
                 }
                 else
                 {
-                    Console.WriteLine("\nIncorrect Input: Empty value is not allowed");
+                    Console.WriteLine("Incorrect Input: Empty value is not allowed");
                     e++;
                 }
             }
-
             if (e == 3)
             {
                 double minNumber = 0.5;
```

Comment:
it looks like this part does its own atomic job. would be great if it is in a separate method